src/utils/designConstants.js

```diff
@@ -0,0 +1,22 @@
+export const BREAKPOINTS = {
```

Comment:
`src/utils/` doesn't quite feel like the right place for constants, at least for a package that is just a library of nouns - I'm wondering about `src/constants/layout.js`, since `designConstants.js` is a little ambiguous, and it would be totally fine to make a new module for colors or whatever other kinds of variables we might want in the future.